src/commands/publish/package.rs

```diff
@@ -30,15 +30,15 @@ impl Package {
         let manifest_path = pkg_path.join("package.json");
         if !manifest_path.is_file() {
             failure::bail!(
-                "Your JavaScript project is missing a `package.json` file; is `{:?}` the \
+                "Your JavaScript project is missing a `package.json` file; is `{}` the \
```

Comment:
i'm a bit confused by this error message (again not introduced in this PR) but maybe we could change it to - "make sure you are running wrangler in the right directory") - then again this probably never gets hit because if they're not in the right directory it probably yelled at them about a wrangler.toml

Reply:
🤷‍♀